frontend/app/work_packages/controllers/details-tab-overview-controller.js

```diff
@@ -137,9 +137,42 @@ module.exports = function($scope,
   (function setupWorkPackageProperties() {
     var otherAttributes = WorkPackagesOverviewService.getGroupAttributesForGroupedAttributes('other', $scope.groupedAttributes);
 
-    angular.forEach($scope.workPackage.props.customProperties, function(customProperty) {
-      this.push(customProperty);
-    }, otherAttributes);
+    function getFormat(workPackage, prop, propName) {
+      return workPackage.props[propName] ?
+              workPackage.props[propName].format :
+              prop.type.toLowerCase();
+    }
+
+    function getValue(workPackage, propName) {
+      if (workPackage.props[propName]) {
+        return workPackage.props[propName].raw;
+      }
+      if (workPackage.embedded[propName]) {
+        return workPackage.embedded[propName].props.id;
+      }
+      return null;
+    }
+
+    function getCustomProperties(workPackage) {
+      return _.compact(_.map(
```

Comment:
@0xF013: Do @houndci and me just not understand your *m4d J4v4Scr1p7 Sk1llz*?

Or is this really an error?